specification/compute/resource-manager/Microsoft.Compute/stable/2021-12-01/disk.json

```diff
@@ -2590,6 +2602,28 @@
         ]
       }
     },
+    "DataAccessAuthMode": {
+      "type": "string",
+      "description": "Add additional AAD authentication requirement when accessing export/upload URL.",
```

Comment:
Additional authentication requirements when exporting or uploading to a disk or snapshot

Reply:
updated